packages/webapp/src/pages/induction/pending-invitations.tsx

```diff
@@ -0,0 +1,66 @@
+import { useQuery } from "react-query";
+
+import { SingleColLayout, Card, PaginationNav, membersStatsQuery } from "_app";
+import { getInductions } from "inductions";
+import { useEffect, useState } from "react";
+import { SpectatorInductions } from "inductions";
+
+const QUERY_INDUCTIONS = "query_inductions";
+const PAGE_SIZE = 10;
```

Comment:
Somethings off with the math around this--an off-by-one somewhere. If there are 10 pending invitations, the "Next Page" button still shows up. 11, it doesn't.

For example, I set this to `5` in my dev environment (there are 5 pending invitations globally) and the "Next Page" button shows up.

![image](https://user-images.githubusercontent.com/7911424/121070978-5d0f2a80-c79d-11eb-8aca-16bb5545c157.png)

Reply:
It'd be worth checking this elsewhere we're paginating too. I *think* I've seen this problem in the Community pagination too.

Reply:
Like this: https://github.com/eoscommunity/Eden/blob/main/packages/webapp/src/pages/members/index.tsx#L107

Reply:
In order to know whether there are more entries, you would need to ask for one more than you can display.

Reply:
exactly. It was a conscious decision to leave this next button there to save one extra request. Do you want me to fix by adding this extra request?

Reply:
Yeah, let’s make the change. It’s not an entirely new request, though, right? Can we just fetch pageSize + 1 per page?

Reply:
(Or maybe it is if we have to fetch each record individually...) I’m not in front of the code atm.

Reply:
Either way, worth it.